docs/DeveloperGuide.md

```diff
@@ -530,6 +531,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
+---
+
+**Use case: UC09 - Add a comment**
```

Comment:
Maybe we can change to edit a comment? since the steps are about editing the comment.